plugin/pkg/admission/gc/gc_admission_test.go

```diff
@@ -134,6 +148,48 @@ func newGCPermissionsEnforcement() (*gcPermissionsEnforcement, error) {
 	return gcAdmit, nil
 }
 
+type neverReturningRESTMapper struct {
+	stopCh <-chan struct{}
```

Comment:
what's the purpose of `stopCh` ? I think that `panicing` in `neverReturningRESTMapper` is enough

Reply:
cruft.  Removed.